### PR TITLE
Example alignment for methods arguments

### PR DESCRIPTION
**Pull request by slawekjaranowski**
Current InteliJ formater - format code in this way - is it correct?

Examples are more clear than words ;-)

### CODE REVIEW

**Review by elharo — COMMENTED** [review]
Which color do you want to paint the bikeshed? :-)

I prefer option 2 (consistent indent) but doing this two different ways for calls and declarations is too confusing. Pick one, or just don't bother with this at all.

**Comment by michael-o** [comment]
> Which color do you want to paint the bikeshed? :-)
> 
> I prefer option 2 (consistent indent) but doing this two different ways for calls and declarations is too confusing. Pick one, or just don't bother with this at all.

Agree here, formatting in `methodNext()` almost everywhere in the code aligns to the column and not 8 spaces.

**Review by michael-o — CHANGES_REQUESTED** [review]
.

**Comment by slawekjaranowski** [comment]
fixed, in both case arguments are aligned

**Review by michael-o — APPROVED** [review]

File: content/apt/developers/conventions/code.apt [hunk 1]
```diff
@@ -126,11 +126,26 @@ public class MyMojo
     // ----------------------------------------------------------------------
     ...
 
+    // multiline arguments should be aligned for method declaration
+    protected void methodOne( String arg1, String arg2,
+                              String arg3, String arg4 )
```
Comment (bmarwell) [inline content/apt/developers/conventions/code.apt:131]: I'd prefer to "chop down" all. Either they fit in one line, or they don't. This is a suggestion to mix, which might result in more diff than needed (when refactoring).

File: content/resources/developers/maven-idea-codestyle.xml [hunk 2]
```diff
@@ -92,6 +92,7 @@ under the License.
     <option name="ELSE_ON_NEW_LINE" value="true" />
     <option name="CATCH_ON_NEW_LINE" value="true" />
     <option name="FINALLY_ON_NEW_LINE" value="true" />
+    <option name="ALIGN_MULTILINE_PARAMETERS_IN_CALLS" value="true" />
```
Comment (bmarwell) [inline content/resources/developers/maven-idea-codestyle.xml:95]: `      <option name="METHOD_PARAMETERS_WRAP" value="5" />`
to chop down parameters on method declarations if too long for a single line.

and maybe
`      <option name="CALL_PARAMETERS_WRAP" value="5" />`
for calls.

At least METHOD_PARAMETERS_WRAP is missing anyway, set it to something like "wrap" (I don't know which number though).

Comment (slawekjaranowski) [reply content/resources/developers/maven-idea-codestyle.xml:95]: I see that Intellij don't export default values ... 😄 
i will try mentioned configuration

File: content/apt/developers/conventions/code.apt (hunk 1 above)
Comment (bmarwell) [reply content/apt/developers/conventions/code.apt:131]: No that is not what I meant.
My idea was: EITHER chop down (one argument per line) OR everything in one line. 

This is example mixed. You have both more than one line and more than one argument per line.

**Review by slachiewicz — APPROVED** [review]

**Comment by slawekjaranowski** [comment]
ok, we are working on new tools for formatting ... so I close it now

**Pull request closed without merging**